.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is excluded from version control. Ensure this is intentional, as it might contain important files that need to be tracked. If it's meant to store user-uploaded files or temporary data, then excluding it is appropriate.